### PR TITLE
Blog app - Add forms

### PR DESCRIPTION
**Pull request by ElsonOtake**
- The current_user method was created in ApplicationController;
- A form to create a Post on behalf of the current_user was created;
- A form to create a comment on behalf of the current_user was created;
- Users to add likes to Posts allowed.

### CODE REVIEW

File: app/controllers/posts_controller.rb [hunk 1]
```diff
@@ -1,12 +1,38 @@
...
+    post.author = current_user
+    respond_to do |format|
+      format.html do
+        if post.save
+          flash[:success] = 'Post was successfully created'
+          redirect_to user_path(current_user)
+        else
+          flash.now[:error] = 'Error: Post could not be saved'
+          render :new, new_user_post_path(current_user)
+        end
```
Comment (mikethreels) [inline app/controllers/posts_controller.rb:34]: - [ ] please have a look at [this article](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/lessons/Controllers_lesson.md#:~:text=Pay%20special%20attention%20to%20the%20strong%20parameters%20description%20in%20Rails%20Guides.) for an example of how to handle params in the controller.

File: app/controllers/comments_controller.rb [hunk 2]
```diff
@@ -0,0 +1,19 @@
+class CommentsController < ApplicationController
```
Comment (mikethreels) [inline app/controllers/comments_controller.rb:1]: - [ ] The comments controller should have new and create actions

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Elson, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- linters are passing.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/controllers/posts_controller.rb (hunk 1 above)
Comment (ElsonOtake) [reply app/controllers/posts_controller.rb:34]: I can’t understand what’s wrong with the way I’m using the parameter. Please, could you be more specific?

**Pull request merged**